en/installation_guide/install_with_vagrant.rst

```diff
@@ -11,11 +11,11 @@ For developers, we provide a Vagrant-powered environment with provisioning so yo
 Install Vagrant
 ---------------
 The project is running on a Vagrant virtual environment built on VirtualBox to be production ready.
+Here is two differents ways to install it.
```

Comment:
I remember we have talked about not giving straight instructions, but more a link to the vagrant page. If I'm right, can you do that please ?